src/lib/backend-client.ts

```diff
@@ -28,6 +28,7 @@ export interface ProjectConfig {
 
 export interface Project {
   name: string;
+  test_collections?: Array<Test>;
   description?: string;
   project_config?: ProjectConfig,
```

Comment:
let's fix that in a different PR 😬 

I think I didn;t use camelCase because (beside forgetting) the json has the value with the `_` in it, so I wasn't sure on how to map it 🤔

Reply:
https://serde.rs/attr-rename.html